diff -urp a/markdown/extensions/wikilinks.py b/markdown/extensions/wikilinks.py
--- a/markdown/extensions/wikilinks.py	2021-04-10 10:13:16.076909532 +0900
+++ b/markdown/extensions/wikilinks.py	2021-04-10 15:45:15.656921356 +0900
@@ -43,7 +43,7 @@ class WikiLinkExtension(Extension):
         self.md = md
 
         # append to end of inline patterns
-        WIKILINK_RE = r'\[\[([\w0-9_ -]+)\]\]'
+        WIKILINK_RE = r'\[\[([\w0-9\.()_ +-]+)\]\]'
         wikilinkPattern = WikiLinksInlineProcessor(WIKILINK_RE, self.getConfigs())
         wikilinkPattern.md = md
         md.inlinePatterns.register(wikilinkPattern, 'wikilink', 75)
